<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>251</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>251</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>361</width>
    <height>251</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Temporizador</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icon.jpg</normaloff>img/icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Time:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="time">
        <property name="minimumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>161</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSet">
        <property name="text">
         <string>Set</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd">
      <property name="minimumSize">
       <size>
        <width>331</width>
        <height>81</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>331</width>
        <height>81</height>
       </size>
      </property>
      <property name="numDigits">
       <number>8</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
